### DISCUSSION TITLE
Coder's Ikigai connect to a buddy: Avni and Yukti

### DISCUSSION THREAD

**Discussion by KaizenGirl1111** [Tech-Phantoms/Maintainers-Program#18] [Application]
Hi, I am Avni Uplabdhee. A 2nd year CSE student at IGTDTUW. I am a front end web developer setting to learn React, strengthening my concepts of JS. I along with my team, aim to build a project 'Connect to a buddy' for our community Coder's Ikigai. In the project, we want to enable a coder from our community to connect and confide his/her doubts, problems with a willing and available coder in the community

LinkedIn - https://www.linkedin.com/in/avni-uplabdhee/
Team mate Yukti- https://www.linkedin.com/in/syukti/
Github - https://github.com/KaizenGirl1111

Plan- https://docs.google.com/document/d/1YXCYMi3zt5kv0mpqqnZHJ4fDPidjsihbw8Ob4Ukn1Pk/edit?usp=sharing

**Comment by imabp** [comment]
This sounds cool. @KaizenGirl1111 , can you tell us more about what exactly is Coder's Ikigai. 

From above context, I got to know, where a user comes, logs in, and ask a question. 
And there will be a newsfeed, showing all the questions asked, and any user can answer that doubt?

Is it something like that? Can you explain more about it.

**Reply by KaizenGirl1111** [reply]
so sorry due to the late reply as our exams are going on!
We want to create a platform for our college that provides mental and physical health support for coders.
We do want to include a service where a person can anonymously ask advice regarding a particular matter bugging him to a person available in the platform!
suggestions are welcome!!
